torch/distributed/_tensor/_utils.py

```diff
@@ -224,3 +224,31 @@ def compute_local_stride(
     return tuple(
         global_stride[i] // stride_divisors[i] for i in range(len(global_stride))
     )
+
+
+def compute_global_padding(
+    global_shape: ShapeType, mesh: DeviceMesh, placements: Sequence[Placement]
+) -> List[int]:
+    """
+    Compute the list of padding needed to make the tensor evenly shardable.
```

Comment:
please give more precise comments: what's the meaning of the return List[int] value.